src/index.js

```diff
@@ -13,9 +13,9 @@ export default options => {
 
   /**
    * Get rooms socket is subscribed to.
-   *
-   * @param {String} id
-   * @param {Function} [fn]
+   * Used for get rooms
```

Comment:
I am totally with u, its just that two lines of comments look redundant, I agree with extending the comment above that line. ;-)